localgraphclustering/SimpleLocal_weighted.py

```diff
@@ -4,7 +4,7 @@
 from .GraphLocal import GraphLocal
 
 def SimpleLocal_weighted(G, ref_nodes,
-                delta: float = 0.3):
+                delta: float = 0.3, relcondflag: bool = True):
```

Comment:
Can't we combine this function with the non-weighted one and use a flag? Essentially all the code is duplicated.